docs/src/main/asciidoc/security-openid-connect-multitenancy.adoc

```diff
@@ -268,14 +268,19 @@ public class CustomTenantResolver implements TenantResolver {
 }
 ----
 
-You can define multiple tenants in your configuration file, just make sure they have a unique alias so that you can map them properly when resolving a tenant from your `TenantResolver` implementation.
+You can define multiple tenants in your configuration file. To map them correctly when resolving a tenant from your `TenantResolver` implementation, ensure each has a unique alias.
+
+However, using a static tenant resolution method, such as configuring tenants in `application.properties` and resolving them with `TenantResolver`, presents limitations. This approach is incompatible with testing the endpoint with Dev Services for Keycloak. The reason is that Dev Services for Keycloak has no knowledge of how requests are mapped to individual tenants. Therefore, it cannot dynamically provide tenant-specific `quarkus.oidc.<tenant-id>.auth-server-url` values. As a result, using `%prod` prefixes with the tenant-specific URLs in `application.properties` is ineffective in tests or development mode.
```

Comment:
```suggestion
However, using a static tenant resolution method, such as configuring tenants in `application.properties` and resolving them with `TenantResolver`, presents limitations. This approach is not suitable for testing the endpoint with Dev Services for Keycloak. The reason is that Dev Services for Keycloak has no knowledge of how requests are mapped to individual tenants. Therefore, it cannot dynamically provide tenant-specific `quarkus.oidc.<tenant-id>.auth-server-url` values. As a result, using `%prod` prefixes with the tenant-specific URLs in `application.properties` is ineffective in tests or development mode.
```

`incompatible` does not seem to work well in this context

Reply:
Updated. Thank you!